### DISCUSSION TITLE
[wiki] non-exists nginx version on wiki

### DISCUSSION THREAD

**Discussion by iDoka** [dani-garcia/vaultwarden#4452] [Ideas]
[This wiki page](https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples) mentioned
```
Nginx - v1.29.0+
```

Meanwhile, based on http://nginx.org/en/download.html the latest stable version of nginx  is **1.24.0**

**Comment by stefan0xC** [comment]
The version refers to the vaultwarden version ([1.29.0](https://github.com/dani-garcia/vaultwarden/releases/tag/1.29.0) or later)

**Comment by iDoka** [comment]
@stefan0xC string before defined as
```
Caddy 2.x
```
on the same wiki page. 

AFAIK v2.x of vaultwarden doesn't exists as well 🤷‍♂️

**Comment by stefan0xC** [comment]
But there is a 2.x of Caddy https://github.com/caddyserver/caddy/releases

**Comment by stefan0xC** [comment]
But I agree that the vaultwarden version does not need to be there as we only support the current release anyway so the old configurations don't need to be listed.
